nucypher/characters/base.py

```diff
@@ -340,6 +337,14 @@ def from_public_keys(cls,
 
         return cls(is_me=False, crypto_power=crypto_power, *args, **kwargs)
 
+    def _set_known_node_class(self, known_node_class, federated_only):
+        if not known_node_class:
+            # Once in a while, in tests or demos, we init a plain Character who doesn't already know about its node class.
+            from nucypher.characters.lawful import Ursula
+            known_node_class = Ursula
```

Comment:
Refresh me -- in what cases can we presume there may exist known nodes who _are not Ursulas_?

Reply:
None, but building ourselves into a box (by not having a class-level namespace used as an interface for the known node class) is an awful idea, right?